qa/tasks/ceph_manager.py

```diff
@@ -1329,7 +1329,7 @@ def tmp(x):
         self.testdir = teuthology.get_testdir(self.ctx)
         self.run_cluster_cmd_prefix = [
             'sudo', 'adjust-ulimits', 'ceph-coverage',
-            f'{testdir}/archive/coverage', 'timeout', '120', 'ceph',
+            f'{self.testdir}/archive/coverage', 'timeout', '120', 'ceph',
```

Comment:
@rishabh-d-dave 
Please move this fixing into the first commit. It's introduced by the first commit and then you're fixing here ?

Reply:
Right. Fixed.